exchange/exchange_test.go

```diff
@@ -3571,6 +3619,14 @@ func TestMakeBidExtJSON(t *testing.T) {
 			expectedBidExt:     ``,
 			expectedErrMessage: "invalid character",
 		},
+		{
```

Comment:
The added test cases look great! Can we please add just one more test case where `prebid.meta` doesn't exist in both `ext` and the bid?

Reply:
Sure. I added a test case where there are no `ext`s. There are also many other tests cases which do not define any `meta` fields.